17_foundations.ipynb

```diff
@@ -1066,15 +1066,15 @@
     "\n",
     "```\n",
     "Image  (3d tensor): 256 x 256 x 3\n",
-    "Scale  (2d tensor):  (1)   (1)  3\n",
+    "Scale  (1d tensor):  (1)   (1)  3\n",
     "Result (3d tensor): 256 x 256 x 3\n",
     "```\n",
     "    \n",
     "However, a 2D tensor of size 256×256 isn't compatible with our image:\n",
     "\n",
     "```\n",
     "Image  (3d tensor): 256 x 256 x   3\n",
-    "Scale  (1d tensor):  (1)  256 x 256\n",
```

Comment:
This is the place with the proper fix

Reply:
Yes - please go ahead and fix that one! :)

Reply:
Ah I see you already did :)